app/views/activities/_feed_entry.html.slim

```diff
@@ -1,10 +1,14 @@
 li class="activity #{activity.kind}"
   = link_to activity.source_url
-    = image_tag(activity.img_url || '/images/screenshot-missing.png')
-  = content_tag :h4, link_to(activity.title, activity.source_url, class: 'title')
+  	- if activity.img_url.present?
```

Comment:
I just did that, but that breaks it locally... this is how it works locally for some reason. do you have an explanation for that?